client/my-sites/plan-features/style.scss

```diff
@@ -1224,6 +1186,7 @@ button.plan-features__scroll-button {
 	}
 }
 
+/* stylelint-disable-next-line no-duplicate-selectors */
```

Comment:
Maybe not ideal, but this is a big CSS file and I didn't want to investigate every possible side effect of combining these two selectors.